orchestrator/order_service.py

```diff
@@ -40,29 +44,158 @@ def initiate_order(self, user_data, payload_dict):
         """
         username = user_data["authorizer"]["claims"]["email"]
         price = payload_dict["price"]
-        order_type = payload_dict["order_type"]
-        order_details = self.manage_create_order(username=username, item_details=payload_dict["item_details"],
-                                                 price=price)
+        order_type = payload_dict["item_details"]["order_type"]
+        item_details = payload_dict["item_details"]
+        group_id = item_details["group_id"]
+        org_id = item_details["org_id"]
+        wallet_address = ""
+        recipient = ""
+        channel_id = ""
+
+        if order_type == OrderType.CREATE_WALLET_AND_CHANNEL.value:
+            recipient = self.get_payment_address_for_org(group_id=group_id, org_id=org_id)
+
+        if order_type == OrderType.FUND_CHANNEL.value:
+            signer = SIGNER_ADDRESS
```

Comment:
Add loggers in order_Service

Reply:
can we move wallet and channel logic to something like wallet_service, and keep order related stuff in order_service